examples/todos/android/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+plugins {
+    id("com.android.application")
+    kotlin("android")
+    kotlin("android.extensions")
+    kotlin("kapt")
+}
+
+android {
+    compileSdkVersion(29)
+    defaultConfig {
+        applicationId = "org.reduxkotlin.example.todos"
+        minSdkVersion(26)
+        targetSdkVersion(29)
+        versionCode = 1
+        versionName = "1.0"
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    buildTypes {
+        getByName("release") {
+            isMinifyEnabled = false
+            proguardFiles(getDefaultProguardFile("proguard-android.txt"), "proguard-rules.pro")
+        }
+        getByName("debug") {
+            // MPP libraries don't currently get this resolution automatically
+            matchingFallbacks = listOf("release")
```

Comment:
probs want to set debugging to true for the debug build. i _think_ it does this by default, but i could be wrong

Reply:
updated